app/views/current_requests/_show.html.slim

```diff
@@ -0,0 +1,98 @@
+.details
+  - if service_request.assigned? and service_request.responded_request_assignment_id?
+    .white-container
+      .text-center.fs20
+        strong.black We`ve found a technician for you
+        .red Please respond as soon as possible
+      .assignments-wrapper.mt20
+        - assignments.each do |assignment|
+          .row
+            .col-sm-3
+              - if assignment.technician
+                = image_tag assignment.technician[:photo] || 'default_avatar_icon.png', class: 'img-circle', width: '70'
+            .col-sm-9
+              = "#{assignment.contractor_account[:name]} can send:"
+              br/
+              = assignment.technician[:name] if assignment.technician
+              .rating-container
+                .star.fullStar
+                .star.fullStar
+                .star.fullStar
+                .star.fullStar
+                .star.fullStar
+              br/
+              = "Can come #{assignment.arrive_eta}"
+          input type='text' placeholder= 'Tap here to enter Promo Code' class= 'w100-pct text-center mt20'
+          .black.text-center.fs16
+            |You must authorize a charge of $10,00
+            .mt10
+              | This covers the technician's arrival and one hour of br/ work to fix or diagnose the problem. Once you br/ accept this conctractor you cannot cancel and your br/ card will be charged when the technician arrives.
+          = link_to 'Accept and authorize charge', '#', remote: true, class: 'btn btn-danger w100-pct text-center mt20'
+          br/
+          = link_to 'Decline', account_service_request_assignment_decline_reasons_path(account, service_request, assignment), remote: true, class: 'btn btn-default w100-pct text-center mt10 with-ajax-loader'
+          p.text-center.fs12.mt10
+            |By Accepting you agree to out 
+            = link_to 'Terms of use', '#'
+          .hidden
+            .bg-red.w100-pct.p10.white.mt20
+                | Authorize Payment
+            .text-center
+              strong Pick a payment method:
+              = link_to 'Decline', '#', class: 'btn btn-default w100-pct text-center mt10'
+              br/
+              = link_to 'Pay $10.00', '#', class: 'btn btn-danger w100-pct text-center mt20'
```

Comment:
@deepali1-aconnexion this hardcoded amount of Diagnostic fee will be changed in future pull-requests?